separatorSup/separator_power_status.db

```diff
@@ -0,0 +1,16 @@
+record(calcout, "$(P)STAT:POWER:_CALC")
+{
+  field(INPA, "$(P)VOLT.VAL CP MS")
+  field(CALC, A>0.1) #PSU ON if voltage across DAQ is >0.1kV
```

Comment:
Should this be testing on the calibrated PSU voltage? I interpreted the ticket such that if the DAQ voltage (in range 0-10V) is lower than 0.1V then the PSU is `off`, else `on`. I'm concerned that a DAQ voltage of 1e-4 will be too small to reliably measure when the PSU voltage is 0.1kV.